Tests/iOS Tests/NSString+Test.h

```diff
@@ -0,0 +1,15 @@
+//
+//  NSString+Test.h
```

Comment:
Call this PNTest just in case there's a name conflict. This category name is too generic.